src/views/TwabRewards/abis/TwabRewards.js

```diff
@@ -0,0 +1,426 @@
+export default [
```

Comment:
Any thoughts on making a `v4-twab-rewards-js` package? Similar to [v4-twab-delegator-js](https://github.com/pooltogether/v4-twab-delegator-js). That example is super slim at the moment and really just [one function](https://github.com/pooltogether/v4-twab-delegator-js/blob/bf60f3b0dc37ab437fe4d4376c911d1fe62b32f0/src/TwabDelegator.ts#L219) but will be nice to isolate the core functionality for different tools even further. It's laid out very similarly to [v4-client-js](https://github.com/pooltogether/v4-client-js). Then integrators will have packages for the full suite of PoolTogether tools. No need to do it yet, but it could be something paired with the claiming.

Reply:
Would likely make sense, especially if there is more functionality there that can be extracted.